bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -174,7 +174,7 @@ parser = argparse.ArgumentParser(usage="",
 
 pycbc.add_common_pycbc_options(parser)
 parser.add_argument("--version", action=pycbc.version.Version)
-parser.add_argument("--template-fit-file", required=True,
+parser.add_argument("--template-fit-file", required=True, nargs='+',
                     help="hdf5 file containing fit coefficients for each"
```

Comment:
Can the description be updated to say this can take multiple files?

Reply:
Thought I'd done that but must have missed it. Added now